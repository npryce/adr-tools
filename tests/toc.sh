adr new First Decision
adr new Second Decision
adr new Third Decision
adr toc
cat doc/adr/0000-contents.md
