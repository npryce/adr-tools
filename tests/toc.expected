adr new First Decision
doc/adr/0001-first-decision.md
adr new Second Decision
doc/adr/0002-second-decision.md
adr new Third Decision
doc/adr/0003-third-decision.md
adr toc
cat doc/adr/0000-contents.md
# Architecture Decision Records

* [1. First Decision](0001-first-decision.md)
* [2. Second Decision](0002-second-decision.md)
* [3. Third Decision](0003-third-decision.md)
